serverless.yml

```
@@ -73,22 +82,42 @@ provider:
 #    - exclude-me-dir/**
 
 functions:
-  set_status_db:
-    handler: handler.handle_set_status_to_db
+  # store status functions
+  set_stores_status_db:
+    handler: store_status_lambda.handle_set_stores_statuses_to_db
+    timeout: 60
+    events:
+      - schedule: rate(2 minutes)
+  get_stores_status_db:
+    handler: store_status_lambda.handle_get_stores_statuses_db
+    events:
+    - http:
+        path: status/stores/db
+        method: get
+  get_num_stores_online:
+    handler: store_status_lambda.handle_get_stores_num_online
+    events:
+    - http:
+        path: status/stores/num
+        method: get
+
+  # service status functions
+  set_services_status_db:
+    handler: website_crawler.handle_set_service_status_to_db
     timeout: 60
     events:
       - schedule: rate(2 minutes)
```

Du kannst dem Event Parameter mitgeben und so hard-coded names im code vermeiden
```suggestion
       - schedule:
          rate: rate(2 minutes)
          input:
            param: 'value'
```